commands/kubernetes.go

```diff
@@ -1679,8 +1679,8 @@ func nodeByName(name string, nodes []*godo.KubernetesNode) (*godo.KubernetesNode
 }
 
 func looksLikeUUID(str string) bool {
-	_, err := uuid.Parse(str)
-	return err == nil
+	r := regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$")
```

Comment:
excellent job - one bit, could we have this `4[a-fA-F0-9]{3}` not hardcoded to version 4 of the standard? maybe [1-4] would be better here?

Reply:
To follow up on this, we're not sure if across all of our different teams we're **only** using UUIDv4s here, or if we're using a combination of UUIDv1s and UUIDv4s. Allowing all versions as valid prevents us from accidentally blocking valid UUIDs.

Reply:
Updated to support versions 1 through 4